workflow/rules/common.smk

```diff
@@ -1046,6 +1046,14 @@ def get_output_dir(wildcards, output):
     return os.path.dirname(output[0])
 
 
+def get_canu_concurrency(threads):
+    if threads > 3:
+        lambda w, threads: int(threads / 4)
```

Comment:
```suggestion
        threads = int(threads / 4)
```

Reply:
True, the lambda is of course not needed here. I took it like that from the rule.
Actually there is an even more concise way by just returning int(threads / 4).
Ill do it this way.

Reply:
Sound good to me 👍